src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityWorkbench.java

```diff
@@ -209,9 +209,11 @@ private AbstractWidgetGroup createWorkbenchTab() {
 
     private AbstractWidgetGroup createItemListTab() {
         WidgetGroup widgetGroup = new WidgetGroup();
+        widgetGroup.addWidget(new LabelWidget(5, 20, "gregtech.machine.workbench.storage_note_1"));
+        widgetGroup.addWidget(new LabelWidget(5, 30, "gregtech.machine.workbench.storage_note_2"));
         CraftingRecipeResolver recipeResolver = getRecipeResolver();
         IItemList itemList = recipeResolver == null ? null : recipeResolver.getItemSourceList();
-        widgetGroup.addWidget(new ItemListGridWidget(2, 17, 9, 6, itemList));
+        widgetGroup.addWidget(new ItemListGridWidget(2, 47, 9, 5, itemList));
```

Comment:
Please move this little bit up so space between this and player inventory is same as space between Crafting Station inventory and player inventory on other tab page.